### PR TITLE
[1LP][RFR] Automating upload/remove image on catalog bundle test

### PR DESCRIPTION
**Pull request by niyazRedhat**
### PRT Run
{{pytest: cfme/tests/services/test_catalog_item.py::test_upload_delete_custom_image_on_bundle --long-running -svvvv }}

### CODE REVIEW

File: cfme/services/catalogs/catalog_items/__init__.py [hunk 1]
```diff
@@ -151,8 +151,9 @@ def is_displayed(self):
 
 
 class DetailsEntitiesCatalogItemView(View):
-    custom_image = FileInput("upload_image")
-    smart_management = SummaryTable("Smart Management")
+    upload_image = FileInput(id="upload_image")
+    upload_buttons = InputButton("commit")
```
Comment (valaparthvi) [inline cfme/services/catalogs/catalog_items/__init__.py:155]: typo: upload_button

File: cfme/tests/services/test_catalog_item.py [hunk 2]
```diff
@@ -624,3 +626,37 @@ def test_add_bundle_in_bundle(appliance, catalog_bundle):
...
+            4. Upload and Remove image should work
+    """
+
+    fs = FTPClientWrapper(cfme_data.ftpserver.entities.others)
+    file_path = fs.download(image)
+
+    view = navigate_to(catalog_bundle, 'Details')
+    view.entities.upload_image.fill(file_path)
+    view.entities.upload_buttons.click()
+    view.flash.assert_message('Custom Image file "logo.png" successfully uploaded')
```
Comment (valaparthvi) [inline cfme/tests/services/test_catalog_item.py:659]:
```suggestion
    view.flash.assert_message(f'Custom Image file {image} successfully uploaded')
```

File: cfme/services/catalogs/catalog_items/__init__.py [hunk 3]
```diff
@@ -151,8 +151,9 @@ def is_displayed(self):
 
 
 class DetailsEntitiesCatalogItemView(View):
-    custom_image = FileInput("upload_image")
-    smart_management = SummaryTable("Smart Management")
```
Comment (digitronik) [inline cfme/services/catalogs/catalog_items/__init__.py:155]: Q: why we are removing `smart_management`?

File: cfme/tests/services/test_catalog_item.py [hunk 4]
```diff
@@ -624,3 +626,37 @@ def test_add_bundle_in_bundle(appliance, catalog_bundle):
     options = view.resources.select_resource.all_options
     assert catalog_bundle.name not in [o.text for o in options]
     view.cancel_button.click()
+
+
+@pytest.mark.meta(automates=[1487056])
+@pytest.mark.parametrize("image", ["logo.png"])
```
Comment (digitronik) [inline cfme/tests/services/test_catalog_item.py:632]: why we are parametrizing if a single known image going to use in test?

File: cfme/services/catalogs/catalog_items/__init__.py (hunk 3 above)
Comment (niyazRedhat) [reply cfme/services/catalogs/catalog_items/__init__.py:155]: thanks for pointing. I forgot to re-add it

**Pull request merged**